x/concentrated-liquidity/incentives.go

```diff
@@ -767,6 +775,7 @@ func (k Keeper) claimAllIncentivesForPosition(ctx sdk.Context, positionId uint64
 
 			// If the claimed incentives are forfeited, deposit them back into the accumulator to be distributed
 			// to other qualifying positions.
+			// NOTE: we cannot hit this logic with current implementation because position gets fully charged at 1nanosecond.
```

Comment:
Im not sure if this is totally true. If we enter and leave in the same block, even if it takes seconds, we go off block time which is the same and therefore position age would be less than the supported uptime.

Reply:
yup removed